docs/howto/custom-management-commands.txt

```diff
@@ -126,52 +126,31 @@ such as :option:`--verbosity` and :option:`--traceback`.
 Management commands and locales
 ===============================
 
-By default, the :meth:`BaseCommand.execute` method deactivates translations
-because some commands shipped with Django perform several tasks (for example,
-user-facing content rendering and database population) that require a
-project-neutral string language.
+By default, management commands are executed with the current active locale.
 
-If, for some reason, your custom management command needs to use a fixed locale,
-you should manually activate and deactivate it in your
-:meth:`~BaseCommand.handle` method using the functions provided by the I18N
-support code::
+If, for some reason, your custom management command should run without an
+active locale (for example, to prevent translated content from being inserted
+into the database), deactivate translations using the ``@no_translations``
+decorator on your :meth:`~BaseCommand.handle` method::
 
-    from django.core.management.base import BaseCommand, CommandError
-    from django.utils import translation
+    from django.core.management.base import BaseCommand, no_translations
 
     class Command(BaseCommand):
         ...
 
+        @no_translations
         def handle(self, *args, **options):
-
-            # Activate a fixed locale, e.g. Russian
-            translation.activate('ru')
-
-            # Or you can activate the LANGUAGE_CODE # chosen in the settings:
-            from django.conf import settings
-            translation.activate(settings.LANGUAGE_CODE)
-
-            # Your command logic here
             ...
 
-            translation.deactivate()
-
-Another need might be that your command simply should use the locale set in
-settings and Django should be kept from deactivating it. You can achieve
-it by using the :data:`BaseCommand.leave_locale_alone` option.
 
-When working on the scenarios described above though, take into account that
-system management commands typically have to be very careful about running in
-non-uniform locales, so you might need to:
+You must also ensure that settings can be imported, as changing the language
```

Comment:
I'm not sure what "You should also ensure that settings can be imported" means, in practice?